articles/libraries/lock/v10/index.md

```diff
@@ -82,15 +81,15 @@ lock.on("authenticated", function(authResult) {
 });
 ```
 
-### Showing Lock
+### 2) Showing Lock
 
 ```js
 document.getElementById('btn-login').addEventListener('click', function() {
   lock.show();
 });
 ```
 
-### Displaying the User's Profile
+### 3) Displaying the User's Profile
 
 ```js
 // Verify that there's a token in localStorage
```

Comment:
I'm a bit confused by the example. Is it retrieving the `profile.nickname` from the access_token? Shouldn't it be the `id_token`?